byron/ledger/formal-spec/properties.tex

```diff
@@ -156,3 +156,76 @@ \subsection{UTxO Properties}
   $$
 
 \end{property}
+
+\subsection{PVBUMP STS Properties}
+\label{sec:pvbump-properties}
+
+Property~\ref{prop:pvbump-no-proposals} states that when no proposals are
+available, the system remains in the same state it started with.
+
+\begin{property}[PVBUMP on no proposals]\label{prop:pvbump-no-proposals}
```

Comment:
In these properties we're not writing an inference rule, so I'd suggest to rephrase this as something like:

![image](https://user-images.githubusercontent.com/175315/58169463-140cdc00-7c91-11e9-8a4b-e49a8927d601.png)

Reply:
Thanks! Done!